### PR TITLE
[stable/node-problem-detector] added support for image digests

### PR DESCRIPTION
**Pull request by thirdeyenick**
## Description

This adds support for image digests to the node-problem-detector chart.

## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] I have read the [contribution instructions](https://github.com/deliveryhero/helm-charts#opening-a-pr), bumped chart version and regenerated the docs
- [ ] Github actions are passing

### CODE REVIEW

**Comment by max-rocket-internet** [comment]
@thirdeyenick fix this file:

```
stable/node-problem-detector/values.yaml
  Error: 61:12 [trailing-spaces] trailing spaces
```

And then run `docker run --rm -v "${PWD}:/helm-docs" jnorwood/helm-docs:v1.11.0 --template-files ./ci/README.md.gotmpl` as per the [contribution instructions](https://github.com/deliveryhero/helm-charts#opening-a-pr) 🙂

**Comment by thirdeyenick** [comment]
> @thirdeyenick fix this file:
> 
> ```
> stable/node-problem-detector/values.yaml
>   Error: 61:12 [trailing-spaces] trailing spaces
> ```
> 
> And then run `docker run --rm -v "${PWD}:/helm-docs" jnorwood/helm-docs:v1.11.0 --template-files ./ci/README.md.gotmpl` as per the [contribution instructions](https://github.com/deliveryhero/helm-charts#opening-a-pr) 🙂

Thank you very much for the hint. I actually ran the `helm-docs` container before, but it removed my manually added `image.digest` entry in the README.md file. I had a look and added the corresponding marker in the `values.yaml` file now.

File: stable/node-problem-detector/README.md [hunk 1]
```diff
@@ -1,42 +1,16 @@
 # node-problem-detector
 
-![Version: 2.3.1](https://img.shields.io/badge/Version-2.3.1-informational?style=flat-square) ![AppVersion: v0.8.12](https://img.shields.io/badge/AppVersion-v0.8.12-informational?style=flat-square)
+![Version: 2.3.2](https://img.shields.io/badge/Version-2.3.2-informational?style=flat-square) ![AppVersion: v0.8.12](https://img.shields.io/badge/AppVersion-v0.8.12-informational?style=flat-square)
 
 This chart installs a [node-problem-detector](https://github.com/kubernetes/node-problem-detector) daemonset. This tool aims to make various node problems visible to the upstream layers in cluster management stack. It is a daemon which runs on each node, detects node problems and reports them to apiserver.
 
 **Homepage:** <https://github.com/kubernetes/node-problem-detector>
 
-## How to install this chart
```
Comment (max-rocket-internet) [inline stable/node-problem-detector/README.md:9]: You can't remove all this 🙂

Comment (thirdeyenick) [reply stable/node-problem-detector/README.md:9]: I did not :-) The helm-docs container run did this automatically. Not sure if there is something missing in the template file for `helm-docs`?

Comment (max-rocket-internet) [reply stable/node-problem-detector/README.md]: I think you're not running it from the repo root as per the instructions?

Comment (thirdeyenick) [reply stable/node-problem-detector/README.md]: Running this in the root dir of the repo clearly helps 🤦  Sorry for the confusion...

**Pull request merged**